app/views/requests/_editrequestitems.html.erb

```diff
@@ -13,7 +13,7 @@
       <% item = Item.find(req_item.item_id) %>
       <tr>
         <td><%= link_to item.unique_name, item_path(item.id) %></td>
-        <td class="text-right">
+        <td class="text-center">
           <a href="#" data-xeditable="true"
```

Comment:
@duckgirl07 I changed these all to the center, I thought it looked better for when I added the serial tags column. If you don't like it or I'm missing some areas feel free to change them back